### ISSUE TITLE
Create sample with ModelState.AddModelError

### ISSUE THREAD

**Issue by Rick-Anderson** [dotnet/AspNetCore.Docs.Samples#110]
In https://github.com/dotnet/AspNetCore.Docs.Samples/tree/main/mvc/models folder

Use in memory DB.
also show how to add a model error to the model, not the property:

```
ModelState.AddModelError(string.Empty, "My error message");
```

see https://github.com/dotnet/AspNetCore.Docs/issues/27861

**Comment by sammychinedu2ky** [comment]
@sammychinedu2ky

**Comment by Rick-Anderson** [comment]
Fixed in #113
